google/cloud/spanner_v1/transaction.py

```diff
@@ -33,6 +33,9 @@
 from google.cloud.spanner_v1._opentelemetry_tracing import trace_call
 from google.cloud.spanner_v1 import RequestOptions
 from google.api_core import gapic_v1
+from google.cloud.spanner_v1._helpers import _retry
```

Comment:
nit: Can you add these imports from helper class at https://github.com/googleapis/python-spanner/pull/934/files#diff-c0a324ccb53381adcd6c782a1d6e4cc18d543a7de1ae49a766977af3ab9bbbf6L20